rest/redirect.go

```diff
@@ -0,0 +1,174 @@
+package rest
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"gopkg.in/ns1/ns1-go.v2/rest/model/redirect"
+)
+
+// RedirectService handles 'redirect' endpoint.
+type RedirectService service
+
+// List returns the configured redirects.
+//
+// NS1 API docs: TBD
```

Comment:
We don't have these docs yet :( Maybe we can add them in a later PR, or delay this one?